### PR TITLE
Translate `the-cascade`

### PR DESCRIPTION
**Pull request by emredevsalot**

### CODE REVIEW

File: temel_bilgiler/html_css/basamaklama.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+}
+~~~
+
+Hem `alert` hem de `warning` sınıflarına sahip bir öğe için basamaklama, kalıtım(bu durumda bulunmuyor) ve özgüllük(iki kural da aynı özgüllüğe sahip) dahil olmak üzere diğer tüm faktörlerin üstünden geçer. `warning` kuralı en son tanımlanan kural olduğu için ve başka hiçbir faktör hangi kuralın uygulanacağını belirleyemediği için öğeye uygulanacak olan kural `warning` kuralıdır.
+
+### Ödev
+
+<div class="lesson-content__panel" markdown="1">
+
+1.  [CSS alıştırmaları repository'mize](https://github.com/TheOdinProject/css-exercises) geri dönün. `foundations` dizinindeki son alıştırma olan `06-cascade-fix` isimli alıştırmayı yapın.
```
Comment (cansirin) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
1.  [Odin Project CSS alıştırmaları repository'sine](https://github.com/TheOdinProject/css-exercises) geri dönün. `foundations` dizinindeki son alıştırma olan `06-cascade-fix` isimli alıştırmayı yapın.
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
+### Giriş
+
+Bir önceki derste temel CSS sözdizimini ve seçicileri ele almıştık. Şimdi seçiciler hakkındaki bilgilerimizi CSS'in *C*'si olan cascade(basamaklama) ile birleştirmenin zamanı geldi.
+
+### Derse Genel Bakış
+
+Bu bölüm, bu derste öğreneceğiniz konulara genel bir bakış içerir.
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
### Derse genel bakış

Bu bölüm, bu derste öğreneceğiniz konuların genel bir özetini içerir.
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+*   Basamaklama ne yapar?
+*   Özgüllük ve CSS seçicilerinin birleştirilmesi.
+*   Kalıtım, belirli özellikleri nasıl etkiler?
+
+### CSS'in Basamaklandırılması
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
### CSS'in basamaklandırılması
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+Bazen birbiriyle çelişen kurallarımız olabilir. Böylece beklenmedik sonuçlarla karşılaşabiliriz. "Ama ben *bu* paragrafların mavi olmasını istemiştim, neden diğer paragraflar gibi kırmızılar?!" Bu durum her ne kadar sinir bozucu olsa da, CSS'in bizim isteğimiz dışında kafasına göre bir şeyler *yapmadığını* anlamak önemlidir. CSS, yalnızca bizim ondan istediğimiz şeyleri yapar. Bu durumun istisnalarından biri, tarayıcı tarafından sağlanan varsayılan stillerdir. Bu varsayılan stiller tarayıcıdan tarayıcıya değişir. Bazı öğelerin diğer öğelerle aralarında büyük bir "boşluk" olmasını ya da butonları o şekilde göstermek için herhangi bir CSS kuralı yazmamamıza rağmen o şekilde görünmesini sağlar.
+
+Dolayısıyla, bunun gibi beklenmedik bir davranışla karşılaşırsanız, bunun nedeni varsayılan stiller, bir özelliğin nasıl çalıştığını anlamamak ya da basamaklama adı verilen bu konsepti anlamamaktır.
+
+Basamaklama, HTML'imize hangi kuralların uygulanacağını belirler.Basamaklamanın bunu belirlemek için kullandığı farklı faktörler vardır. Şimdi bu faktörlerden üçünü inceleyeceğiz. Bunların üstünden geçmek, CSS'ten nefret ettiğiniz anlardan kaçınmanıza yardımcı olacaktır.
+
+
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+#### Özgüllük
+
+Daha özgül bir CSS tanımlaması, ondan daha az özgül olanlardan önceliklidir. Önceki derste ele aldığımız satır içi stiller, seçicilere kıyasla en yüksek özgüllüğe sahipken, her seçici türünün bir tanımlamanın ne kadar özgül olduğuna katkıda bulunan kendi özgüllük düzeyi vardır. Diğer seçiciler de özgüllüğe katkıda bulunurlar. Ancak biz yalnızca şu ana kadar üstünde durduklarımıza odaklanacağız:
+
+1. ID seçiciler (en özgül seçici)
+2. Sınıf seçiciler
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
2. Sınıf(class) seçiciler
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+
+Yukarıdaki örnekte, her iki kural da yalnızca sınıf seçicileri kullanmaktadır. Ancak 2. kural daha özgüldür. Çünkü daha fazla sınıf seçicisi kullanmaktadır, bu nedenle `color: red;` tanımlaması öncelikli olacaktır.
+
+Şimdi, birkaç şeyi değiştirelim:
+
+~~~html
+<!-- index.html -->
+
+<div class="main">
+  <div class="list" id="subsection"></div>
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
  <div class="list" id="subsection">Mavi yazı</div>
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+}
+
+/* kural 2 */
+.main .list {
+  color: red;
+}
+~~~
+
+Yukarıdaki örnekte, kural 2'de ID seçici sayısından daha fazla sınıf seçicisi vardır. Ancak ID, sınıftan öncelikli olduğu için kural 1 daha özgüldür. Bu durumda, `color: blue;` tanımlaması öncelikli olacaktır.
+
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md:78]:
````suggestion

Biraz daha karmaşık:

```html
<!-- index.html -->

<div class="main">
  <div class="list" id="subsection">Sarı arkaplanlı kırmızı yazı</div>
</div>
```

````

File: temel_bilgiler/html_css/basamaklama.md [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+}
+
+.child {
+  color: blue;
+}
+~~~
+
+`parent` öğesi bir ID ile seçildiği için daha yüksek bir özgüllüğe sahip olmasına rağmen, `child` öğesine `color: blue` stili uygulanır. Çünkü bu tanımlama doğrudan onu hedef alır. Ancak ebeveynden gelen `color: red` tanımlaması yalnızca miras olarak alınır.
+
+#### Kural Sırası
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
#### Kural sırası
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 9]
```diff
@@ -0,0 +1,219 @@
...
+    - Nasıl isterseniz öyle biçimlendirebilirsiniz. Ancak harici CSS yöntemini kullanmalısınız(bu pratik ve ilerisi için). Ayrıca önceki derste bahsedilen özelliklerden birkaçını(renk, arka plan rengi, tipografi özellikleri, vb.) kullanmaya çalışmalısınız. Çeşitli özelliklerin ne işe yaradıklarını anlamak için onlara biraz zaman ayırın. Şimdilik *iyi* görünmesi hakkında hiç endişelenmeyin. Bu yalnızca pratik yapmak ve CSS yazmaya alışmak içindir. Özgeçmişinizde gösterecek bir şey yapmak için değil.
+    - Henüz `font-family` özelliği için özel bir fontun nasıl kullanılacağından bahsetmedik. Bu nedenle şimdilik kullanılacak genel font ailelerinin bir listesi için [CSS Fontlar](https://www.w3schools.com/Css/css_font.asp) ve internet güvenli fontların bir listesi için [CSS İnternet Güvenli Fontlar](https://www.w3schools.com/cssref/css_websafe_fonts.asp) adreslerine bir göz atın. İnternet güvenli fontlar, her bilgisayarda veya cihazda yüklü olan fontlardır(ancak yine de yedek olarak genel bir font ailesi eklediğinizden emin olun).
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
    - Henüz `font-family` özelliği için özel bir fontun nasıl kullanılacağından bahsetmedik. Bu nedenle şimdilik kullanılacak genel font ailelerinin bir listesi için [CSS Fonts](https://www.w3schools.com/Css/css_font.asp) ve internet güvenli fontların bir listesi için [CSS Web Safe Fonts(İnternet Güvenli Fontlar)](https://www.w3schools.com/cssref/css_websafe_fonts.asp) adlı ingilizce makalelere bir göz atın. İnternet güvenli fontlar, her bilgisayarda veya cihazda yüklü olan fontlardır(ancak yine de yedek olarak genel bir font ailesi eklediğinizden emin olun).
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 10]
```diff
@@ -0,0 +1,219 @@
...
+
+</div>
+
+### Bilgi Ölçme
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
### Bilgi ölçme
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 11]
```diff
@@ -0,0 +1,219 @@
...
+
+Bu bölüm, dersi anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayıp yönlendirdiği materyali inceleyin.
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
Bu bölüm, bu dersi kendi kendinize anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayın ve bağlantılı olduğu materyali gözden geçirin.
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 12]
```diff
@@ -0,0 +1,219 @@
...
+
+*   <a class="knowledge-check-link" href="#high-specificity-class-type">Sınıf seçici kullanan bir kural ile üç adet tip seçici kullanan bir kural arasında hangisi daha yüksek özgüllüğe sahiptir?</a>
+
+### Ek Kaynaklar
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
### Ek kaynaklar
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 13]
```diff
@@ -0,0 +1,219 @@
...
+*   [Yazı Tipi Ailesini Değiştirme](https://www.digitalocean.com/community/tutorials/how-to-load-and-use-custom-fonts-with-css#finding-and-loading-a-font-file-from-a-hosted-service) özel yazı tipleri kullanmaya yönelik birkaç farklı yaklaşımı açıklamaktadır.
+*   Kevin Powell'dan [CSS Özgüllükleri](https://www.youtube.com/watch?v=c0kfcP_nD9E) çeşitli özgüllük örneklerini inceleyip özgüllükle boğuşmaktan kaçınmak için bazı tavsiyeler vermektedir.
+*   [CSS Özgüllük Hesaplayıcı](https://specificity.keegan.st/) kendi seçicilerinizi doldurmanıza ve bunların özgüllüklerinin hesaplanıp görselleştirilmesini sağlar.
+*   [Mozilla CSS Özellikleri Referansı](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference) belirli bir CSS özelliğinin miras alınıp alınmadığını öğrenmek için kullanılabilir. Herhangi bir öğenin sayfasına gidip **Formal Definition** bölümü içinde **Inherited** alanına göz atmanız yeterlidir. Örneğin [`color` özelliğinin sayfasına](https://developer.mozilla.org/en-US/docs/Web/CSS/color#formal_definition) gidersek miras alınıp alınmadığını görebiliriz.
+*   [CSS Basamaklama Üzerine İnteraktif Scrim.](https://scrimba.com/scrim/c9gwmnAR)
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
 - [The CSS Cascade(CSS Basamaklama)](https://wattenberger.com/blog/css-cascade) adlı ingilizce makale, CSS kurallarının uygulanmasını etkileyen diğer faktörler hakkında biraz daha ayrıntılı bilgi veren harika, etkileşimli bir kaynaktır.
 - [Yazı Tipi Ailesini Değiştirme](https://www.digitalocean.com/community/tutorials/how-to-load-and-use-custom-fonts-with-css#finding-and-loading-a-font-file-from-a-hosted-service) hakkında olan bu ingilizce makale, özel yazı tipleri kullanmaya yönelik birkaç farklı yaklaşımı açıklamaktadır.
 - Kevin Powell'dan [CSS Specificity](https://www.youtube.com/watch?v=c0kfcP_nD9E) adlı bu ingilizce video, çeşitli özgüllük örneklerini inceleyip özgüllükle boğuşmaktan kaçınmak için bazı tavsiyeler vermektedir.
 - [CSS Specificity Calculator](https://specificity.keegan.st/) adlı bu ingilizce website, kendi seçicilerinizi doldurmanıza ve bunların özgüllüklerinin hesaplanıp görselleştirilmesini sağlar.
 - [Mozilla CSS Properties Reference](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference) adlı ingilizce kaynak, belirli bir CSS özelliğinin miras alınıp alınmadığını öğrenmek için kullanılabilir. Herhangi bir öğenin sayfasına gidip **Formal Definition** bölümü içinde **Inherited** alanına göz atmanız yeterlidir. Örneğin [`color` özelliğinin sayfasına](https://developer.mozilla.org/en-US/docs/Web/CSS/color#formal_definition) gidersek miras alınıp alınmadığını görebiliriz.
 - [CSS Basamaklama üzerine ingilizce bir interaktif Scrim.](https://scrimba.com/scrim/c9gwmnAR)
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 14]
```diff
@@ -0,0 +1,219 @@
...
+    - Henüz `font-family` özelliği için özel bir fontun nasıl kullanılacağından bahsetmedik. Bu nedenle şimdilik kullanılacak genel font ailelerinin bir listesi için [CSS Fontlar](https://www.w3schools.com/Css/css_font.asp) ve internet güvenli fontların bir listesi için [CSS İnternet Güvenli Fontlar](https://www.w3schools.com/cssref/css_websafe_fonts.asp) adreslerine bir göz atın. İnternet güvenli fontlar, her bilgisayarda veya cihazda yüklü olan fontlardır(ancak yine de yedek olarak genel bir font ailesi eklediğinizden emin olun).
+
+</div>
+
+### Bilgi Ölçme
+
+Bu bölüm, dersi anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayıp yönlendirdiği materyali inceleyin.
+
+*   <a class="knowledge-check-link" href="#high-specificity-class-type">Sınıf seçici kullanan bir kural ile üç adet tip seçici kullanan bir kural arasında hangisi daha yüksek özgüllüğe sahiptir?</a>
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
- <a class="knowledge-check-link" href="#high-specificity-class-type">Sınıf seçici kullanan bir kural ile üç adet tip seçici kullanan bir kural arasında hangisi daha yüksek özgüllüğe sahiptir?</a>
```

File: temel_bilgiler/html_css/basamaklama.md (hunk 9 above)
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
2.  "HTML Temelleri" bölümünde alıştırma olarak oluşturduğunuz tarif sayfasını hatırlıyor musunuz? Oldukça *sade* görünüyor değil mi? Hadi bunu biraz CSS ekleyerek düzeltelim!
   - Nasıl isterseniz öyle biçimlendirebilirsiniz. Ancak harici CSS yöntemini kullanmalısınız(bu pratik ve ilerisi için). Ayrıca önceki derste bahsedilen özelliklerden birkaçını(renk, arka plan rengi, tipografi özellikleri, vb.) kullanmaya çalışmalısınız. Çeşitli özelliklerin ne işe yaradıklarını anlamak için onlara biraz zaman ayırın. Şimdilik *iyi* görünmesi hakkında hiç endişelenmeyin. Bu yalnızca pratik yapmak ve CSS yazmaya alışmak içindir. Özgeçmişinizde gösterecek bir şey yapmak için değil.
   - Henüz `font-family` özelliği için özel bir fontun nasıl kullanılacağından bahsetmedik. Bu nedenle şimdilik kullanılacak genel font ailelerinin bir listesi için [CSS Fontlar](https://www.w3schools.com/Css/css_font.asp) ve internet güvenli fontların bir listesi için [CSS İnternet Güvenli Fontlar](https://www.w3schools.com/cssref/css_websafe_fonts.asp) adreslerine bir göz atın. İnternet güvenli fontlar, her bilgisayarda veya cihazda yüklü olan fontlardır(ancak yine de yedek olarak genel bir font ailesi eklediğinizden emin olun).
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 15]
```diff
@@ -0,0 +1,219 @@
...
+~~~
+
+Hem `alert` hem de `warning` sınıflarına sahip bir öğe için basamaklama, kalıtım(bu durumda bulunmuyor) ve özgüllük(iki kural da aynı özgüllüğe sahip) dahil olmak üzere diğer tüm faktörlerin üstünden geçer. `warning` kuralı en son tanımlanan kural olduğu için ve başka hiçbir faktör hangi kuralın uygulanacağını belirleyemediği için öğeye uygulanacak olan kural `warning` kuralıdır.
+
+### Ödev
+
+<div class="lesson-content__panel" markdown="1">
+
+1.  [Odin Project CSS alıştırmaları repository'sine](https://github.com/TheOdinProject/css-exercises) geri dönün. `foundations` dizinindeki son alıştırma olan `06-cascade-fix` isimli alıştırmayı yapın.
+    Not: Alıştırmaların çözümleri her alıştırmanın `solution` klasöründe bulunur.
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
1. [Odin Project CSS alıştırmaları repository'sine](https://github.com/TheOdinProject/css-exercises) geri dönün. `foundations` dizinindeki son alıştırma olan `06-cascade-fix` isimli alıştırmayı yapın.


    Not: Alıştırmaların çözümleri her alıştırmanın `solution` klasöründe bulunur.
```

File: temel_bilgiler/html_css/basamaklama.md [hunk 16]
```diff
@@ -0,0 +1,219 @@
...
+Bir önceki derste temel CSS sözdizimini ve seçicileri ele almıştık. Şimdi seçiciler hakkındaki bilgilerimizi CSS'in *C*'si olan cascade(basamaklama) ile birleştirmenin zamanı geldi.
+
+### Derse Genel Bakış
+
+Bu bölüm, bu derste öğreneceğiniz konulara genel bir bakış içerir.
+
+*   Basamaklama ne yapar?
+*   Özgüllük ve CSS seçicilerinin birleştirilmesi.
+*   Kalıtım, belirli özellikleri nasıl etkiler?
+
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
- Basamaklama ne yapar?
- Özgüllük ve CSS seçicilerinin birleştirilmesi.
- Kalıtım, belirli özellikleri nasıl etkiler?

```

File: temel_bilgiler/html_css/basamaklama.md [hunk 17]
```diff
@@ -0,0 +1,219 @@
...
+/* kural 1 */
+.subsection {
+  color: blue;
+}
+
+/* kural 2 */
+.main .list {
+  color: red;
+}
+~~~
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
````suggestion
```html
<!-- index.html -->

<div class="main">
  <div class="list subsection"></div>
</div>
```

```css
/* kural 1 */
.subsection {
  color: blue;
}

/* kural 2 */
.main .list {
  color: red;
}
```
````

File: temel_bilgiler/html_css/basamaklama.md [hunk 18]
```diff
@@ -0,0 +1,219 @@
...
+
+Yukarıdaki örnekte, her iki kural da yalnızca sınıf seçicileri kullanmaktadır. Ancak 2. kural daha özgüldür. Çünkü daha fazla sınıf seçicisi kullanmaktadır, bu nedenle `color: red;` tanımlaması öncelikli olacaktır.
+
+Şimdi, birkaç şeyi değiştirelim:
+
+~~~html
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
````suggestion
```html
````

File: temel_bilgiler/html_css/basamaklama.md [hunk 19]
```diff
@@ -0,0 +1,219 @@
...
+/* kural 1 */
+#subsection {
+  color: blue;
+}
+
+/* kural 2 */
+.main .list {
+  color: red;
+}
+~~~
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
````suggestion
```
```css
/* kural 1 */
#subsection {
  color: blue;
}

/* kural 2 */
.main .list {
  color: red;
}
```
````

File: temel_bilgiler/html_css/basamaklama.md [hunk 20]
```diff
@@ -0,0 +1,219 @@
...
+#subsection {
+  background-color: yellow;
+  color: blue;
+}
+
+/* kural 2 */
+.main #subsection {
+ color: red;
+}
+~~~
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
````suggestion
```css
/* kural 1 */
#subsection {
  background-color: yellow;
  color: blue;
}

/* kural 2 */
.main #subsection {
 color: red;
}
```
````

File: temel_bilgiler/html_css/basamaklama.md [hunk 21]
```diff
@@ -0,0 +1,219 @@
...
+/* rule 1 */
+.class.second-class {
+  font-size: 12px;
+}
+
+/* rule 2 */
+.class .second-class {
+  font-size: 24px;
+}
+~~~
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
````suggestion
```css
/* rule 1 */
.class.second-class {
  font-size: 12px;
}

/* rule 2 */
.class .second-class {
  font-size: 24px;
}
```
````

File: temel_bilgiler/html_css/basamaklama.md [hunk 22]
```diff
@@ -0,0 +1,219 @@
...
+/* rule 1 */
+.class.second-class {
+  font-size: 12px;
+}
+
+/* rule 2 */
+.class > .second-class {
+  font-size: 24px;
+}
+~~~
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
````suggestion
```css
/* rule 1 */
.class.second-class {
  font-size: 12px;
}

/* rule 2 */
.class > .second-class {
  font-size: 24px;
}
```
````

File: temel_bilgiler/html_css/basamaklama.md [hunk 23]
```diff
@@ -0,0 +1,219 @@
...
+/* rule 1 */
+* {
+  color: black;
+}
+
+/* rule 2 */
+h1 {
+  color: orange;
+}
+~~~
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
````suggestion
```css
/* rule 1 */
* {
  color: black;
}

/* rule 2 */
h1 {
  color: orange;
}
```
````

File: temel_bilgiler/html_css/basamaklama.md [hunk 24]
```diff
@@ -0,0 +1,219 @@
...
+/* styles.css */
+
+#parent {
+  color: red;
+}
+
+.child {
+  color: blue;
+}
+~~~
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
````suggestion
```html
<!-- index.html -->

<div id="parent">
  <div class="child"></div>
</div>
```
```css
/* styles.css */

#parent {
  color: red;
}

.child {
  color: blue;
}
```
````

File: temel_bilgiler/html_css/basamaklama.md [hunk 25]
```diff
@@ -0,0 +1,219 @@
...
+/* styles.css */
+
+.alert {
+  color: red;
+}
+
+.warning {
+  color: yellow;
+}
+~~~
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
````suggestion
```css
/* styles.css */

.alert {
  color: red;
}

.warning {
  color: yellow;
}
```
````

File: temel_bilgiler/html_css/basamaklama.md [hunk 26]
```diff
@@ -0,0 +1,240 @@
...
+Biraz daha karmaşık:
+
+```html
+<!-- index.html -->
+
+<div class="main">
+  <div class="list" id="subsection">Sarı arkaplanlı kırmızı yazı</div>
+</div>
+```
+
```
Comment (hueypov) [inline temel_bilgiler/html_css/basamaklama.md]:
```suggestion
```

**Comment by hueypov** [comment]
@emredevsalot first of all, so sorry for notification bomb, I had to do it. Since this PR is a little bit outdated, I have to update from original file. Since most of the changes are from updates, I didn't want to make you busy for this. 

Other than that, thank you for all your efforts on this project!!. Everything looks perfect

**Pull request merged**